src/tooltip/dataToMarkup.js

```diff
@@ -12,73 +12,121 @@ export default (data/*: Object */) => {
   const targetEnd = _data.targetEnd || _data.tEnd || _data.tend;
   const queryStart = _data.queryStart || _data.qStart || _data.qstart;
   const queryEnd = _data.queryEnd || _data.qEnd || _data.qend;
+
   let coordinates;
-  if (_data.end) {
-    const length = _data.end - _data.start;
-    const aliStart = _data.aliStart || _data.start;
-    const aliEnd = _data.aliEnd || _data.end;
+
+  if (_data.start && _data.end) {
+    coordinates = `${_data.start} - ${_data.end}`;
+
+    if (_data.aliStart && _data.aliEnd) {
+      coordinates += ` (alignment region ${_data.aliStart} - ${_data.aliEnd})`;
+    }
+  }
+
+  let modelMatch;
+
+  if (_data.modelStart) {
     let offset = 0;
-    if (aliStart - _data.start > 0) {
-      offset = (aliStart - _data.start) / length * 100;
+
+    if (_data.modelStart > 1) {
+      offset = (100 * (_data.modelStart - 1)) / _data.modelLength;
     }
-    coordinates = `
-      ${_data.start}<span style="width: 100px;" class="domain">
+
+    modelMatch = `
+      1<span style="width: 100px;" class="domain">
         <span class="alignment" style="
-          width: ${(aliEnd - aliStart) * 100 / length}px;
+          width: ${
+            (100 * (_data.modelEnd - _data.modelStart + 1)) / _data.modelLength
```

Comment:
Not sure, but should this be 

```suggestion
            (100 * (_data.modelEnd - (_data.modelStart + 1))) / _data.modelLength
```